src/test/resources/examples/AsyncAwait.js

```diff
@@ -3,9 +3,8 @@ async function f(){}
 class C {
     async method(){}
     static async method1(){}
-    async static #method2(){}
```

Comment:
should a function name prefixed with a `#` work anyway?

Reply:
This one is a change from https://github.com/antlr/grammars-v4/pull/3530 . But, yeah, I think '#" before a method in class makes it private, somehow.  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties#private_methods